script/release

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
I highly recommend using at least some amount of [Bash Strict Mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/).